libraries/AP_Scripting/lua_bindings.h

```diff
@@ -10,3 +10,5 @@ int lua_get_i2c_device(lua_State *L);
 int AP_HAL__I2CDevice_read_registers(lua_State *L);
 int lua_get_CAN_device(lua_State *L);
 int lua_get_CAN_device2(lua_State *L);
+int lua_dirlist(lua_State *L);
+int lua_removefile(lua_State *L);
```

Comment:
There is a `os.remove()` in core lua we could enable.

Reply:
or at least you could bind this function to that.

Reply:
we will try to enable os, with just remove and rename

Reply:
done